packages/components/custom-elements-manifest.config.js

```diff
@@ -225,6 +225,26 @@ export default {
       }
     },
 
+    // Since these changes from shoelace (https://github.com/shoelace-style/shoelace/pull/2255), where they removed the @property from "value" and changed the "defaultValue" and "value" handling internally
```

Comment:
I am not sure if we should leave the new "defaultValue" field in here. I think it might be confusing. Consider the following react code:

```tsx
        <SynSelect
          id="select-role"
          label="Current position"
          name="role"
          required
          defaultValue="frontend"
          value={formData.role}
        >
          <SynOptgroup label="Developers">
            <SynOption value="backend">Backend Developer</SynOption>
            <SynOption value="frontend">Frontend Developer</SynOption>
          </SynOptgroup>
          <SynOptgroup label="Other">
            <SynOption value="lead">Team Lead</SynOption>
            <SynOption value="other">Other (please specify in comment section below)</SynOption>
          </SynOptgroup>
        </SynSelect>
```

In this case, I am assigning both the value and the defaultValue. The defaultValue will only be visible when using a form reset, otherwise the value property will always be preferred.

Reply:
Talked about this, defaultValue was always there, so no need to hide it.